### DISCUSSION TITLE
When will security support for Puma 5.x end?

### DISCUSSION THREAD

**Discussion by ReidDelfeld** [puma/puma#3312] [Q&A] [answered]
When will security support for Puma 5.x end? I would like to know so I can plan when we should move to the latest version of Puma. Thanks.

**Comment by dentarg** [comment, marked as answer]
I think when Puma v7 is released (and when that happens is not decided, follow the repo and you may get a hunch)

We have this document https://github.com/puma/puma/blob/master/SECURITY.md, it isn't stated in the document explicitly but I think the policy is to support the two last major releases 

@nateberkopec should have more info on this

**Comment by nateberkopec** [comment]
Yup - that's correct! We don't have solid plans for a 7.0 release yet but it could be tomorrow, could be 2 years from now.
